tasks/README.md

```diff
@@ -313,3 +314,52 @@ Returns all tasks of the requested user.
     - **Content:** `{ 'statusCode': 404, 'error': 'Not Found', 'message': 'Task doesn't exist' }`
   - **Code:** 500
     - **Content:** `{ 'statusCode': 500, 'error': 'Internal Server Error', 'message': 'An internal server error occurred' }`
+
+
+## **PATCH /tasks/:id/status**
+
+- **Params**  
+  _Required:_ `id=[string]`
+
+- **Headers**  
+  Content-Type: application/json
+
+- **Body**
+
+  ```
+  {
+    status: <new-status>
+    percentCompleted: <number>
+  }
+  ```
+
+- **Cookie**  
+  rds-session: `<JWT>`
+
+- **Success Response:**
+  - **Code**: 200
+
+```
+{
+  message: 'Task updated successfully!'
+  userStatus?: {
+    status : "String",
+    message : "String"
+    data: {
+      "previousStatus"?: "String",
+      "currentStatus"?: "String",
+      "futureStatus"?: "String"
+    }
+  }
+}
+```
+
+- **Error Response:**
+  - **Code:** 401
+    - **Content:** `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'User can not be authenticated' }`
```

Comment:
can we change this error message "User can not be authenticated'"  to *Unauthenticated User*